<!DOCTYPE html>
<html>

<head>
  	<title>Research Experience</title>
	<link href="assets/css/style.css" rel="stylesheet">
	<link rel="shortcut icon" type="image/x-icon" href="favicon.ico?">
</head>

<body>

<div id="nav-placeholder"></div>

<script src="//code.jquery.com/jquery.min.js"></script>
<script>
$.get("./navigation.html", function(data){
    $("#nav-placeholder").replaceWith(data);
});
</script>


</body>
	
<p class="ex2">I have undertaken research projects both as part of my academic course, as well as during a summer holiday, in order to further my experience of practical work and scientific research, as this is something I would like to pursue in my future career. Personally, I feel the ability to better understand and make discoveries relating to disease mechanisms provides an additional reward to pure clinical practice.</p>

<hr />

<h4 class="ex2">Summer Research Project — Karam Teixeira Group (Department of Genetics, University of Cambridge, CB2 3EH): 01/07/2019 – 28/08/2019</h4>
<p class="ex2">I completed a <a href="fkt2019.html">research project</a> in this group in order to further my interest in molecular genetics and germline stem cell development. My project was investigating the phenotype of Hybrid Dysgenesis in D. melanogaster and looking at the degree of germline stem cell rescue in flies with varying alleles of mnk. <em>Mnk</em> (also known as <em>chk2</em>, with the human homolog CHEK2) is a tumour suppressor kinase involved in the DNA damage response pathway, that acts downstream of ATM, and can induce DNA repair, or can progress to induce apoptosis (seen in dysgenic flies). Various mutants of mnk have been created by previous work in the lab. In parallel, I performed experiments to characterize the germline development phenotype of dysgenic progeny carrying new suppressor mutations (previously identified through an EMS screen). I was investigating the distribution of double strand DNA breaks induced during meiosis using anti-pH2Av, as well as its correlation with germ cell rescue. </p>
<p class="ex2">Using a GFP transgene fused to the p53 gene, I was able to conclude that these new suppressor mutations were not acting on or downstream of p53, as I had expected. I thoroughly enjoyed my time in the lab, as not only did I greatly develop my technical research skills, but also allowed me to gain key skills such as protocol optimisation, and further my ability to make experimental hypotheses. My work was very kindly supported by a Undergraduate Studentship from the Pathological Society of Great Britain and Northern Ireland, as well as a Caius Medical Association Scholarship, as well as an honorary Gonville &amp; Caius College Summer Internship Grant.</p>
<hr />

<h4 class="ex2">Natural Sciences Tripos, Part II Genetics Research Project — Ferguson-Smith/Hamilton Group (Department of Genetics, University of Cambridge, CB2 3EH): 20/01/2021 – 19/03/2021</h4>
<p class="ex2">I completed a <a href="nst2gn.html">research project</a> in this group as part of my Part II course in Genetics. This project was chosen due to my interests in the developmental aspects of disease, and I was able to gain a new perspective on this through the field of structural bioinformatics and RNA modelling. My project was investigating the structural similarity between specific small nucleolar RNAs (snoRNAs) in splicing, as they have been implicated in alternative splicing, as well as in the pathogenesis of Prader-Willi Syndrome, an imprinting disorder resulting from maternal disomy of regions of Chromosome 15. My project has shown that there is conservation of the U4 dependent spliceosome between <em>Mus musculus</em> and <em>Homo sapiens</em>, as well as showing that there is structural similarity between a subset of snoRNAs from clusters within the <em>Dlk1-Dio3 </em>and <em>Snrpn </em>imprinted regions. This structural similarity is seen in both humans and mice, and thus lends support to my hypothesis that these snoRNAs might hijack or interfere with the spliceosome in some way, affecting the process of splicing.</p>
<p class="ex2">While the learning curve was rather steep in this project, as I had minimal previous bioinformatics experience, I found it thoroughly rewarding, learning many new skills and further improving my ability to tackle scientific hypotheses. Working directly with Dr. Russell Hamilton to arrange and plan (as well as undertake) this project has allowed me to formulate my own research questions, and propose my own research questions. I continue to be involved with Dr. Hamilton's work on snoRNAs, and shall be  continuing my research into these structurally acting snoRNAs, within the context of cardiovascular disease and restenosis (first suggested by <a href="https://doi.org/10.3390/ijms20235975">Nossent et al., 2019</a> and <a href="https://doi.org/10.1093/cvr/cvy309">Håkansson et al., 2019</a>).</p>

<hr />

<h4 class="ex2">Research Skills gained:</h4>
<ul>
  <li>Immunofluorescence staining</li>
  <li>Confocal microscopy</li>
  <li>Illumina DNA/RNA Sequencing</li>
  <li>Agilent Bioanalyzer 2100</li>
  <li>Running gel electrophoresis experiments</li>
  <li>General fly husbandry techniques</li>
</ul>
<ul>
  <li>Sequence conservation analysis/alignment</li>
  <li>DNA/RNA 2D structure prediction</li>
  <li>DNA/RNA 2D structural distance analysis</li>
  <li>RNA 3D structure prediction</li>
  <li>Biochemical 3D structure visualisation</li>
  <li>Biochemical 3D structure alignment and comparison<br>
  </li>
</ul>
<ul>
  <li>Competent in <i>R</i></li>
  <li>Competent in <i>bash</i></li>
</ul>
</body>
</html>
